src/styles/_catalog.scss

```diff
@@ -9,14 +13,34 @@
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
-    height: 300px;
+    width: $width-card;
+    height: $height-card;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
+    margin-bottom: $card-padding;
   }
 
   &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+    margin-right: $card-padding;
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+  }
+
+  @media (min-width: 768px) and (max-width: 1023px) {
```

Comment:
after that you will need only 3 media @media screen and (min-width: 488px) @media screen and (min-width: 768px) @media screen and (min-width: 1024px)